llvm/test/ThinLTO/X86/distributed_indexes.ll

```diff
@@ -48,6 +48,13 @@
 ; RUN: llvm-dis %t1.bc.thinlto.bc -o - | FileCheck %s --check-prefix=DIS
 ; DIS: aliasee: null
 
+; function-import pass crashed when alias is imported but aliasee doesn't.
```

Comment:
I think the comment and TODO can now be removed?

Reply:
Thanks for the catch! Removed the outdated comments.